server/server.go

```diff
@@ -46,24 +75,31 @@ func main() {
 
 	r := internal.NewServer(storageClient, containerName)
 
-	udpPort := 5688
-	dTLSPort := 5689
-	udpAddr := fmt.Sprintf("%s:%d", *address, udpPort)
-	dtlsAddr := fmt.Sprintf("%s:%d", *address, dTLSPort)
+	// old ports
+	go launchServer(flagValues, ports.oldUdpPort, ports.oldDtlsPort, r)
+
+	// new ports
+	launchServer(flagValues, ports.newUdpPort, ports.newDtlsPort, r)
+
+}
+
+func launchServer(flagValues flags, udpPort int, dtlsPort int, r *mux.Router) {
```

Comment:
Move to new module

Reply:
Thanks for this, yes, moved it out like here --> [7938d5c](https://github.com/NordicPlayground/nordic-developer-academy-coap/pull/4/commits/7938d5c15d0677b445c8282a8681c343a7505864)